app/assets/javascripts/models/data/service_options.js.coffee

```diff
@@ -119,7 +118,12 @@ ns.ServiceOptions = do (ko, edsc = @edsc, KnockoutModel = @edsc.models.KnockoutM
                 if !accessMethodMatched && matches[1] == jsonObj.collection_id && parseInt(matches[2], 10) == index
                   accessMethodMatched = true
                   @loadForm(true) if jsonObj.type == 'service'
-                  ko.applyBindingsToNode(echoformContainer, {echoform: this})
+                  # After the binding is complete, toggle select elements to trigger validation lest they be pruned
+                  $.when(ko.applyBindingsToNode(echoformContainer, echoform: this)).done ->
```

Comment:
@mightynimble  - Echoforms builds itself so generically that I'm not seeing an obvious way to limit the selects to those only within the reprojection section.  The only way I can think of atm is to update echoforms to add a generic class name around the reprojection section to mark it, and then use that as an anchor to somehow narrow down the scope of the toggle.  In other words  $('.echoforms-element-select').each will become something like  $('.projectionSection.echoforms-element-select').each (syntax might be off).

The con is that this would take a moment or two to figure out and then retesting and such - so, it wouldn't be done probably until the first part of next week (being this is late Thursday and I have a higher priority ticket).  Do we want to delay this current fix in order to _potentially_ save a second on very large forms?

Further - while this particular bug is specific to reprojection params, it isn't difficult to imagine that other select params that we're not aware of may also suffer this same bug and so will require toggling.  The bonus of doing all of them is it makes sure that all of them have been validated. 

@mreese84 - what do you think?